content/en/blog/2023/ambient-merged-istio-main/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Istio Ambient Service Mesh Merged to Istio’s Main Branch"
+description: A significant milestone for ambient mesh.
+publishdate: 2023-02-27
+attribution: "John Howard (Google), Lin Sun (Solo.io)"
+keywords: [istio,ambient]
+---
+
+Istio ambient service mesh was [launched in Sept 2022](/blog/2022/introducing-ambient-mesh/) in an experimental branch, introducing a new data plane mode for Istio without sidecars. Through collaboration with the Istio community, across Google, Solo.io, Microsoft, Intel, Aviatrix, Huawei, IBM and others, we are excited to announce that Istio ambient mesh has graduated from the experimental branch and merged to Istio’s main branch! This is a significant milestone for ambient mesh, paving the way for releasing ambient in Istio 1.18 and becoming the default in Istio’s future releases.
```

Comment:
```suggestion
Istio ambient service mesh was [launched in Sept 2022](/blog/2022/introducing-ambient-mesh/) in an experimental branch, introducing a new data plane mode for Istio without sidecars. Through collaboration with the Istio community, across Google, Solo.io, Microsoft, Intel, Aviatrix, Huawei, IBM and others, we are excited to announce that Istio ambient mesh has graduated from the experimental branch and merged to Istio’s main branch! This is a significant milestone for ambient mesh, paving the way for releasing ambient in Istio 1.18 and becoming the default data plane mode in Istio’s future releases.
```

Reply:
@howardjohn what do you think of this suggestion? I am a little hesitate to commit it because we don't change ambient to be default data plane mode, it is choice by user whether they choose sidecar or ambient.

Reply:
@linsun, without my suggested change it's unclear what `becoming the default` means. Does it mean `available in the default profile`? Should clarify one way or another or just remove the `becoming ...` part of the sentence.

Reply:
Hi @frankbu - What I really wanted to mean is ambient profile becoming the default profile in Istio's future releases.

Reply:
Should I just say the default profile?